buildbot/master/files/config/master.cfg

```diff
@@ -8,10 +8,10 @@ from passwords import SLAVE_PASSWORD, CHANGE_PASSWORD
 from passwords import HOMU_BUILDBOT_SECRET, GITHUB_STATUS_TOKEN
 
 
-LINUX_SLAVES = ["servo-linux1", "servo-linux2", "servo-linux3"]
+LINUX_SLAVES = ["servo-linux{}".format(i) for i in xrange(1, 7)]
 MAC_SLAVES = ["servo-mac2", "servo-mac3", "servo-mac4", "servo-macpro1"]
 CROSS_SLAVES = ["servo-linux-cross1", "servo-linux-cross2"]
-WINDOWS_SLAVES = ["servo-windows1"]
+WINDOWS_SLAVES = ["servo-windows1", "servo-windows2"]
```

Comment:
Might as well do the same for `CROSS_SLAVES` and `WINDOWS_SLAVES`...`MAC_SLAVES` is probably fine as a hardcoded list for now considering their current volatility.